contour/how-to-guides/configuring-contour-with-loadbalancer-tls-termination.hbs.md

```diff
@@ -101,7 +99,9 @@ Follow these steps to configure your Tanzu Application Platform:
      - name: overlay-contour-envoy
     ```
 
-    Where `ARN` is the ARN recored in [Create a TLS certificate in ACM](#create-tls).
+   Where `ARN` is the ARN recored in [Create a TLS certificate in ACM](#create-tls).
+
+   >**Note** If plan to use Classic LoadBalancer remove `contour.envoy.service.aws.LBType: aws`
```

Comment:
@mhoshi-vm @KauzClay Did you mean to say "contour.envoy.service.aws.LBType: **nlb**" instead of "contour.envoy.service.aws.LBType: **aws**" here?

Reply:
Sorry correct ... my typo